swingset/src/main/java/com/nqadmin/swingset/utils/SSDataGridScreenHelper.java

```diff
@@ -539,6 +539,9 @@ public void updateScreen() {
 			
 			// UPDATE SELECTION CRITERIA FOR ANY OTHER SSDBCombos
 			updateSSDBComboBoxes();
+			
```

Comment:
Per the JavaDoc above this method, we indicate that setParentID() should be called prior to calling updateScreen(). I'm concerned that calling setDefaultValues() at the end of updateScreen() could overwrite values in a row unexpectedly.